etl/steps/export/explorers/faostat/latest/global_food.py

```diff
@@ -0,0 +1,514 @@
+waste,${food_singular} waste in supply chains,"The quantity that is lost or wasted in supply chains through poor handling, spoiling, lack of refrigeration and damage from the field to retail. It does not include consumer waste.",Waste in supply chains,,false,LineChart,waste_in_supply_chain__tonnes,true,The FAO apply a methodological change from the year 2010 onwards.
+waste,Per capita ${food_singular_lower} waste in supply chains,"The quantity that is lost or wasted in supply chains through poor handling, spoiling, lack of refrigeration and damage from the field to retail. It does not include consumer waste.",Waste in supply chains,,true,LineChart,waste_in_supply_chain__kg__per_capita,true,The FAO apply a methodological change from the year 2010 onwards.
+food_balances,${food_plural} used for direct human food,"The quantity that is allocated for direct consumption as human food, rather than allocation to animal feed or industrial uses.",Allocated for human food,,false,LineChart,food__tonnes,true,The FAO apply a methodological change from the year 2010 onwards.
+food_balances,"${food_plural} used for direct human food, per capita","The quantity that is allocated for direct consumption as human food, rather than allocation to animal feed or industrial uses.",Allocated for human food,,true,LineChart,food__kg__per_capita,true,The FAO apply a methodological change from the year 2010 onwards.
+food_balances,${food_plural} used for industrial uses,"The quantity that is allocated to industrial uses such as biofuel, pharmaceuticals or textile products.",Allocated to industrial uses,,false,LineChart,other_uses__tonnes,true,The FAO apply a methodological change from the year 2010 onwards.
+food_balances,"${food_plural} used for industrial uses, per capita","The quantity that is allocated to industrial uses such as biofuel, pharmaceuticals or textile products.",Allocated to industrial uses,,true,LineChart,other_uses__kg__per_capita,true,The FAO apply a methodological change from the year 2010 onwards.
+animal_feed,${food_plural} used for animal feed,The quantity that is allocated to feed for livestock.,Allocated to animal feed,,false,LineChart,feed__tonnes,true,The FAO apply a methodological change from the year 2010 onwards.
+animal_feed,"${food_plural} used for animal feed, per capita",The quantity that is allocated to feed for livestock.,Allocated to animal feed,,true,LineChart,feed__kg__per_capita,true,The FAO apply a methodological change from the year 2010 onwards.
+"""
+
+
+def run():
+    #
+    # Load inputs.
+    #
+    # Load the template for the explorer tsv file.
+    template = Template(TEMPLATE_CONTENT)
+    # Load the foods data.
+    foods_df = pd.read_csv(StringIO(FOODS_CONTENT), index_col="slug", dtype=str)
+    # Load the views per food data.
+    views_df = pd.read_csv(StringIO(VIEWS_PER_FOOD_CONTENT), dtype=str)
+
+    # Determine whether the script is executed on staging or production.
+    if OWID_ENV.name == "production":
```

Comment:
This is nice, I'll move it right before sending the TSV to Admin API (but merge as it is).